OpenRA.Mods.Common/AI/Squad.cs

```diff
@@ -77,6 +76,12 @@ public bool TargetIsValid
 			get { return Target.IsValidFor(Units.FirstOrDefault()) && !Target.Actor.HasTrait<Husk>(); }
 		}
 
+		public bool TargetIsInVisible
+		{
+			get { return Target.Type == TargetType.Actor && !TargetActor.HasTrait<FrozenUnderFog>()
+				 && !Units.FirstOrDefault().Owner.Shroud.IsTargetable(TargetActor); }
```

Comment:
Can you squash those fixup commits? We see it is resolved by CI checks and the diff comments hiding.